README.md

```diff
@@ -76,105 +64,185 @@ Take note of the following properties for use in the next section:
+            objectType: secret        # object types: secret, key or cert
+            objectVersion: ""         # [OPTIONAL] object versions, default to latest if empty
+          - |
+            objectName: key1
+            objectAlias: ""
+            objectType: key
+            objectVersion: ""
+      resourceGroup: "rg1"            # [REQUIRED for version < 0.0.4] the resource group of the KeyVault
+      subscriptionId: "subid"         # [REQUIRED for version < 0.0.4] the subscription ID of the KeyVault
+      tenantId: "tid"                 # the tenant ID of the KeyVault
+
+  ```
+
+  | Name                   | Required | Description                                                     | Default Value |
+  | -----------------------| -------- | --------------------------------------------------------------- | ------------- |
+  | provider               | yes      | specify name of the provider                                    | ""            |
+  | usePodIdentity         | no       | specify access mode: service principal or pod identity          | "false"       |
+  | useVMManagedIdentity   | no       | [__*available for version > 0.0.4*__] specify access mode to enable use of VM's managed identity    |  "false"|
+  | userAssignedIdentityID | no       | [__*available for version > 0.0.4*__] the user assigned identity ID is required for VMSS User Assigned Managed Identity mode  | ""       |
+  | keyvaultName           | yes      | name of a Key Vault instance                                    | ""            |
+  | cloudName              | no       | [__*available for version > 0.0.4*__] name of the azure cloud based on azure go sdk (AzurePublicCloud,AzureUSGovernmentCloud, AzureChinaCloud, AzureGermanCloud)| "" |
+  | objects                | yes      | a string of arrays of strings                                   | ""            |
+  | objectName             | yes      | name of a Key Vault object                                      | ""            |
+  | objectAlias            | no       | [__*available for version > 0.0.4*__] specify the filename of the object when written to disk - defaults to objectName if not provided | "" |
+  | objectType             | yes      | type of a Key Vault object: secret, key or cert                 | ""            |
+  | objectVersion          | no       | version of a Key Vault object, if not provided, will use latest | ""            |
+  | resourceGroup          | no      | [__*required for version < 0.0.4*__] name of resource group containing key vault instance            | ""            |
+  | subscriptionId         | no      | [__*required for version < 0.0.4*__] subscription ID containing key vault instance                   | ""            |
+  | tenantId               | yes      | tenant ID containing key vault instance                         | ""            |
+
+#### Provide Identity to Access Key Vault
```

Comment:
This step is referenced in `Create your own SecretProviderClass Object`. Instead would it be better to make this the second step, right after `Create a new Azure Key Vault resource or use an existing one`?

Reply:
All 4 modes reference updates to the `SecretProviderClass` resource or the application deployment yaml, which would make more sense _after_ the user has gone thru the `Create your own SecretProviderClass Object` section. WDYT?

Reply:
Yeah, makes sense!